src/User/UserContainer.jsx

```diff
@@ -1,18 +1,24 @@
 import React, { useState, useEffect } from "react";
 import { Switch, Route, useParams } from "react-router-dom";
 import ProfilePage from "./ProfilePage";
-import { getUserById } from "../api.js";
+import { getUserById, getFavouritedConferenceById } from "../api.js";
 
 const UserContainer = ({ token }) => {
   const [user, setUser] = useState(null);
+  const [favouriteConferences, setFavouriteConferences] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
   const [error, setError] = useState(false);
   const { id } = useParams();
 
-  const fetchData = async userId => {
+  const fetchData = async (userId, conferenceId) => {
     setIsLoading(true);
 
     try {
+      const favouriteConferences = await getFavouritedConferenceById(
```

Comment:
I think either I'm confused or this should be userId? I assume you're trying to get a list of conferences a user has expressed interest in? But currently you're trying to get a list of conferences with conference ID